pkg/tests/api/correlations/correlations_read_test.go

```diff
@@ -0,0 +1,319 @@
+package correlations
```

Comment:
there are 3 skipped tests in this file, all related to what i think is some accesscontrol quirk and the way the test env is set up as viewer or editors do not get `datasource:read` permission.

Reply:
yep, GE tests fail as expected.

Reply:
fixed.